src/layers/core/point-density-hexagon-layer/point-density-hexagon-layer.js

```diff
@@ -102,6 +135,13 @@ export default class PointDensityHexagonLayer extends Layer {
     });
   }
 
+  _onPercentileChange() {
+    const [lower, upper] = [this.props.lowerPercentile, this.props.upperPercentile];
```

Comment:
This seems like a slighly surprising way to decompose. Why not use the standard idiom?
```
const {lowerPercentile, upperPercentile} = this.props;
...getCountRangeFromPercentile(this.state.sortedCounts, [lowerPercentile, upperPercentile])
```